lib/api/core/models/repositories/repository.js

```diff
@@ -6,28 +6,39 @@ var
 
 /**
  * @param {Kuzzle} kuzzle
- * @param options
  * @constructor
  */
-function Repository (kuzzle, options) {
-  this.collection = options.collection;
-  this.index = options.index;
-  this.ttl = options.ttl || kuzzle.config.repositories.cacheTTL;
-  this.ObjectConstructor = options.ObjectConstructor;
-  this.writeLayer = options.writeLayer;
+function Repository (kuzzle) {
+  if (arguments.length === 0) { return; }
```

Comment:
Nitpicking: This does not comply with NPM coding rules. Moreover, is this check necessary, since `Repository` objects are only used internally (risk of wrong arguments list is low)